### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by jacky309**

### CODE REVIEW

File: src/ipc/ipc-common/ipc-serialization.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**********************************************************************
+**
+** Copyright (C) 2018 Luxoft Sweden AB
```
Comment (weimerb) [inline src/ipc/ipc-common/ipc-serialization.cpp]: 2019

File: src/ipc/ipc-common/ipc-serialization.h [hunk 2]
```diff
@@ -0,0 +1,291 @@
+/**********************************************************************
+**
+** Copyright (C) 2018 Luxoft Sweden AB
```
Comment (weimerb) [inline src/ipc/ipc-common/ipc-serialization.h]: 2019

File: src/ipc/dbus/DBusIPCProxy.h [hunk 3]
```diff
@@ -160,6 +160,8 @@ class DBusIPCProxy : public IPCProxyBase<InterfaceType>, protected DBusIPCProxyB
 
 public:
     typedef const char *MemberIDType;
```
Comment (weimerb) [inline src/ipc/dbus/DBusIPCProxy.h]: Nitpick: mixture between typedef and using declarations - if you are at it, could you switch to "using"?

File: src/ipc/dbus/DBusIPCServiceAdapter.h [hunk 4]
```diff
@@ -151,6 +151,8 @@ class DBusIPCServiceAdapter : public DBusIPCServiceAdapterBase
 
 public:
     typedef ServiceType TheServiceType;
```
Comment (weimerb) [inline src/ipc/dbus/DBusIPCServiceAdapter.h]: Same here

File: src/ipc/ipc-common/ipc-serialization.cpp (hunk 1 above)
Comment (jacky309) [reply src/ipc/ipc-common/ipc-serialization.cpp]: done with next commits

File: src/ipc/dbus/DBusIPCProxy.h (hunk 3 above)
Comment (jacky309) [reply src/ipc/dbus/DBusIPCProxy.h]: done with next commits

File: src/ipc/dbus/DBusIPCServiceAdapter.h (hunk 4 above)
Comment (jacky309) [reply src/ipc/dbus/DBusIPCServiceAdapter.h]: done with next commits

File: src/ipc/ipc-common/ipc-serialization.h (hunk 2 above)
Comment (jacky309) [reply src/ipc/ipc-common/ipc-serialization.h]: done with next commits

**Pull request merged**